solvers/amigo.py

```diff
@@ -26,7 +26,9 @@
 
 
 class Solver(BaseSolver):
-    """Two loops solver."""
```

Comment:
why not use the `reference` object in the solver, like here: https://github.com/benchopt/benchmark_lasso/blob/a6d131381f0751fd59c663765b441df52c0a219b/solvers/cyanure.py#L16C1-L20C6 ?

Reply:
It seems that when using using docstring, the reference appears in the solver description in the html result
![image](https://github.com/benchopt/benchmark_bilevel/assets/77896657/262f01c3-3942-44ac-b9d6-86db0ae2ae4b)

while it is not the case when using `references`
![image](https://github.com/benchopt/benchmark_bilevel/assets/77896657/4e974ff9-ee36-41c5-9587-1c81cc226de7)